In [9]:
from ultralytics import YOLO
import time

In [3]:
ZenoC = YOLO("yolo11n-obb.pt")
start =time.time()
test = ZenoC("Akshay Kumar_0.jpg",imgsz=320, workers =8)
end = time.time()
test[0].show()
print(f"Total Time:{end-start:.3f} seconds")


image 1/1 F:\RNN based Object detection and Anomaly Classification surveillance System\Akshay Kumar_0.jpg: 320x224 (no detections), 173.4ms
Speed: 8.5ms preprocess, 173.4ms inference, 12.3ms postprocess per image at shape (1, 3, 320, 224)
Total Time:0.398 seconds


In [18]:

ZenoC.export(format="openvino",dynamic=False,half=False,int8=True,opset=12,data='calibration.yaml')

Ultralytics 8.3.196  Python-3.12.7 torch-2.8.0+cpu CPU (Intel Core i5-8250U 1.60GHz)

PyTorch: starting from 'yolo11n-obb.pt' with input shape (1, 3, 1024, 1024) BCHW and output shape(s) (1, 20, 21504) (5.5 MB)

OpenVINO: starting export with openvino 2025.3.0-19807-44526285f24-releases/2025/3...
OpenVINO: collecting INT8 calibration images from 'data=calibration.yaml'

ERROR OpenVINO: export failure 9.2s: Dataset 'calibration.yaml' images not found, missing path 'F:\Study\RNN based Object detection and Anomaly Classification surveillance System\calibration\images'
Note dataset download directory is 'F:\RNN based Object detection and Anomaly Classification surveillance System\datasets'. You can update this in 'C:\Users\hp\AppData\Roaming\Ultralytics\settings.json'


FileNotFoundError: Dataset 'calibration.yaml' images not found, missing path 'F:\Study\RNN based Object detection and Anomaly Classification surveillance System\calibration\images'
Note dataset download directory is 'F:\RNN based Object detection and Anomaly Classification surveillance System\datasets'. You can update this in 'C:\Users\hp\AppData\Roaming\Ultralytics\settings.json'

image 1/1 F:\RNN based Object detection and Anomaly Classification surveillance System\Akshay Kumar_0.jpg: 320x224 (no detections), 34.2ms
Speed: 0.9ms preprocess, 34.2ms inference, 0.6ms postprocess per image at shape (1, 3, 320, 224)
Total Time:0.120 second